src/styles/_reset.scss

```diff
@@ -0,0 +1,11 @@
+body {
+  margin: 0;
+}
+
+h1 {
+  margin: 0;
+}
```

Comment:
same styles could be combined using `,` operator
```
body,
h1 {...}
```

Reply:
fixed